### PR TITLE
Augmented Arguments

### PR DESCRIPTION
**Pull request by zhiayang**

### CODE REVIEW

File: src/main/java/chopchop/logic/edit/EditDescriptor.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// EditDescriptor.java
+
+package chopchop.logic.edit;
+
+public class EditDescriptor {
+}
```
Comment (seowalex) [inline src/main/java/chopchop/logic/edit/EditDescriptor.java:6]: Why is there an empty class?

Comment (zhiayang) [reply src/main/java/chopchop/logic/edit/EditDescriptor.java:6]: It's just to act as a base class for the 3 sub-edit-descriptors, which was necessary cos of the way i wrote `parseEditCommand`

Comment (seowalex) [reply src/main/java/chopchop/logic/edit/EditDescriptor.java:6]: The `editType` field and accessor seems to be shared by all 3 class, maybe consider putting it in the base class?

Comment (zhiayang) [reply src/main/java/chopchop/logic/edit/EditDescriptor.java:6]: 5head

**Pull request merged**